xls/modules/zstd/BUILD

```diff
@@ -17,11 +17,11 @@
 load("@rules_hdl//place_and_route:build_defs.bzl", "place_and_route")
```

Comment:
consider adding https://google.github.io/xls/bazel_rules_macros/#xls_dslx_fmt_test to enforce DSLX formatting.